cli/internal/prune/prune.go

```diff
@@ -142,12 +143,18 @@ func (p *prune) prune(opts *opts) error {
 		}
 	}
 	workspaces := []turbopath.AnchoredSystemPath{}
-	targets := []interface{}{opts.scope}
-	internalDeps, err := ctx.TopologicalGraph.Ancestors(opts.scope)
+	targets := []string{opts.scope}
+	internalDeps, err := ctx.WorkspaceGraph.Ancestors(opts.scope)
 	if err != nil {
 		return errors.Wrap(err, "could find traverse the dependency graph to find topological dependencies")
 	}
-	targets = append(targets, internalDeps.List()...)
+
+	// Use for loop so we can coerce to string
+	// .List() returns a list of interface{} types, but
+	// we know they are strings.
+	for _, dep := range internalDeps.List() {
+		targets = append(targets, dep.(string))
+	}
```

Comment:
Calling this out. I'm not 100% sure this is safe (although compiler seems ok with it). Is `interface{}` like TS `any`?

Reply:
It'll panic, but if we're storing non-strings as vertices in our graph, we have big issues so a panic feels right.